content/en/docs/reference/using-api/deprecation-guide.md

```diff
@@ -49,6 +49,13 @@ RuntimeClass in the **node.k8s.io/v1beta1** API version will no longer be served
 * All existing persisted objects are accessible via the new API
 * No notable changes
 
+#### PodSecurityPolicy {#psp-v125}
```

Comment:
my initial version kept this page limited to things with clear replacements. I was going to see what shook out for PSP this release, and if cronjob graduates to v1 as expected in 1.21, update this section for both of them after that.

Reply:
also, PSP deprecation is getting released with 1.21, so I'd hold this at least until then

Reply:
I'm OK with merging this into the live website before v1.21 released, provided that:
- the deprecation in v1.21 is planned out, including how we communicate it
- we are confident beyond any reasonable doubt of the deprecation timeline, we are committing ourselves to make exactly the changes we're documenting here
- we make this update in tandem with the broader piece about communiction

Reply:
See PR https://github.com/kubernetes/website/pull/27369, _Add blog post describing PSP deprecation and next steps_.

Reply:
(now merged)